hack/rhcos-sas/rhcos-sas.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	azstorage "github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/openshift/installer/pkg/types"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/env"
```

Comment:
These need to be updated to the ARO-Installer versions, not ARO-RP

Reply:
Creating a new task to address this, as this needs to be addressed in go.mod and the entire package.